modules/persistent-storage-local-create-cr-manual.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-create-cr-manual_{context}"]
+= Provisioning local volumes without the Local Storage Operator
+
+Local volumes cannot be created by dynamic provisioning. Instead, persistent volumes can be created by defining the persistent volume (PV) in an object definition. The local volume provisioner looks for any file system or block volume devices at the paths specified in the defined resource.
+
+[IMPORTANT]
+====
+Manual provisioning of PVs includes the risk of potential data leaks across PV reuse when PVCs are deleted.
+The Local Storage Operator is recommmended for automating the life cycle of devices when provisioning local PVs.
+====
+
+.Prerequisites
+
+* Local disks are attached to the {product-title} nodes.
+
+.Procedure
+
+. Define the persistent volume (PV). Create a file, such as `example-pv-filesystem.yaml` or `example-pv-block.yaml`, with the `PersistentVolume` object definition. This resource must define the nodes and paths to the local volumes.
++
+[NOTE]
+====
+Do not use different storage class names for the same device. Doing so will create multiple persistent volumes (PVs).
```

Comment:
Since you have PV defined right above, you might be able to just say PVs in the note

Reply:
Yes, good catch!